src/NovaDependencyContainer.php

```diff
@@ -110,4 +111,11 @@ protected function fillAttributeFromRequest(NovaRequest $request, $requestAttrib
             $field->fill($request, $model);
         }
     }
+
+
+    public function setWatchable($name, $path=null)
```

Comment:
I absolutely like the idea for a `setWatchable` method, though I would primarily use this internally, and only make it available up front as a last resort to set the watcher on a different attribute, if for example a Field or changes were made in Nova this package isn't supporting yet.